### ISSUE TITLE
Options to set both skips simultaneously and to disable the \sectionbreak

### ISSUE THREAD

**Issue by gusbrs** [michal-h21/sectionbreak#1]
@michal-h21 If I may, I have a couple of suggestions for `sectionbreak`.

The first one would be to add an option `skip` which would set both `preskip` and `postskip` to the same value. It makes sense, as it saves options in loading the package, specially as I assume most of the time both skips will be the same (they are by default).

The second would be an option to disable `\sectionbreak` (and Co.). It may be useful in some circumstances to do so, without removing the markup in the document. Of course, one could always just `\renewcommand{\sectionbreak}{}`, but once it has become a package, an option would be more elegant, I think.

**Comment by michal-h21** [comment]
@gusbrs of course, these seems like nice ideas. I've added `disable` and `skip` options. Could you please try if that works for you?

**Comment by gusbrs** [comment]
@michal-h21 Thank you!

I tried things here, as you asked.

First, in the `disable` option, shouldn't `\asterism` also be emptied?

Second, the skip option does work fine, but perhaps more robustness in its workings would be welcome. I would expect, as you use `\ProcessKeyvalOptions*` that options take effect in sequence. For example, I'd expect:

```
\usepackage[skip=1em,preskip=5em]{sectionbreak}
```
to give me `postskip=1em` and `preskip=5em`, but the first `skip` seems to override the explicit call to `preskip` that comes later. Would it be too complicated for options to be processed in sequence? (I'm not sure it's worth it, though, please consider).

Finally, how about `\RequirePackage{kvoptions}` instead of `\usepackage{kvoptions}`?

**Comment by michal-h21** [comment]
@gusbrs thanks for the feedback!

>  First, in the `disable` option, shouldn't `\asterism` also be emptied?

I think `\asterism` should be used mainly as an argument for the `\sectionbreak`, so it shouldn't be necessary to empty it.

> Would it be too complicated for options to be processed in sequence? (I'm not sure it's worth it, though, please consider).

It wasn't hard to fix that :) It should work now.

**Comment by gusbrs** [comment]
@michal-h21 Thank you once again! It looks all good now. Already in use here. :-)

As to `\asterism` I had got wrong its intended use. You are correct it shouldn't be necessary to empty it then.

**Comment by michal-h21** [comment]
@gusbrs great, I should post it to CTAN soon :)
